src/main/java/uk/gov/hmcts/reform/em/stitching/domain/Bundle.java

```diff
@@ -225,4 +231,39 @@ public String toString() {
                 + this.hasTableOfContents + ", hasCoversheets=" + this.hasCoversheets + ", hasFolderCoversheets="
                 + this.hasFolderCoversheets + ")";
     }
+
+    @Transient
+    public Integer getSubtiles(SortableBundleItem container, Map<BundleDocument, File> documents) {
+        return container
+                .getSortedItems().flatMap(SortableBundleItem::getSortedDocuments)
+                .map(i -> {
+                    try {
+                        return load(documents.get(i)).getDocumentCatalog().getDocumentOutline();
+                    } catch (IOException e) {
+                        e.printStackTrace();
+                    }
+                    return null;
+                })
+               .filter(o->o!=null)
+               .filter(o->o.getFirstChild()!=null)
+               .mapToInt(o -> o!=null? getItemsFromOutline.apply(o) : 0).sum();
+    }
+
+    @Transient
+    Function<PDDocumentOutline,Integer> getItemsFromOutline =(outline) ->{
```

Comment:
Any reason this isn't a private method?

Reply:
its a function called only by the new lambda hence i didnt specify it private . Made it private now as anyway no other class or method needs to use it